src/inline-liveevents/index.js

```diff
@@ -0,0 +1,49 @@
+import { write } from '../_shared/js/dom';
+import { getIframeId, getWebfonts, resizeIframeHeight } from '../_shared/js/messages';
+import { getApiBaseUrl } from '../_shared/js/dev';
+import { formatPrice, formatDuration } from '../_shared/js/utils';
+import { generatePicture } from '../_shared/js/capi-images';
+
+let container = document.getElementsByClassName('adverts__body')[0];
+
+let params = new URLSearchParams();
+params.append('t', '[%EventID%]');
+
+getIframeId()
+.then(({ host, preview }) => fetch(`${getApiBaseUrl(host, preview)}/commercial/api/liveevent.json?${params}`, {mode: 'cors'}))
+.then(response => response.json())
+.then(createAdvert)
+.then(html => Promise.all([getWebfonts(), write(() => container.innerHTML = html)]))
+.then(resizeIframeHeight)
+
+function createAdvert(event) {
+
+  let lastFew = event.ratioTicketsLeft <= 0.1 ? '<div class="advert__meta advert__meta--scarcity">Last few tickets remaining</div>' : '';
+
+  return `<a class="blink advert advert--inline advert--brand" href="%%CLICK_URL_ESC%%${event.eventUrl}" data-link-name="merchandising-liveEvents-superHigh-${event.eventId}-${event.name}">
```

Comment:
Gladly, but do they not use this for tracking? I thought there were a bunch of old-looking `data-link` attributes that have stuck around because merchandise/duffet use them for reporting...?